nixos/modules/services/desktops/gnome3/evince.nix

```diff
@@ -0,0 +1,37 @@
+# Evince.
+
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.gnome3.evince = {
+
+      enable = mkEnableOption ''
+        Enables the user systemd and dbus services for Evince, the GNOME
```

Comment:
I think `mkEnableOption` prefixes "Whether to enable" and adds a period at the automatically.

Reply:
Ooops, yes indeed it does.